### DISCUSSION TITLE
Show two save actions besides each other instead of dropdown

### DISCUSSION THREAD

**Discussion by hagealex** [Laravel-Backpack/community-forum#454] [General] [answered]
Hello everyone!

I'm working on a kind of CMS. My clients are able to write content. This content can either be saved as a draft or published.

Right now I have two actions for the save button:
1. Save as draft
2. Save and publish

The default behavior of Backpack is to show all other actions besides the default one in a dropdown/select.

![grafik](https://user-images.githubusercontent.com/21688684/227581470-b94a40d2-e1a1-448a-8fbb-3761a5bb3f28.png)

My question: Is it possible to show two save buttons beside each other instead of in a select/dropdown?

The intention behind this is to make both actions visible in one place.

Does someone know if this is possible?

**Comment by hagealex** [comment]
An additional note. What I want to achieve looks like this (just played around with the DOM in the browser inspektor):
![grafik](https://user-images.githubusercontent.com/21688684/227582666-5e046e16-2f4c-4f1c-a40d-69579e280045.png)

**Comment by jcastroa87** [comment, marked as answer]
Hi @hagealex 

To add a save action you need to follow instruction in our [documentation](https://backpackforlaravel.com/docs/5.x/crud-save-actions#addsaveaction-array-saveaction)

If you additionally want to customize button view you will need to override /backpack/crud/src/resources/views/crud/inc/form_save_buttons.blade.php because the only way to show today is "btn-group".

Cheers

**Reply by hagealex** [reply]
Thanks for the hint! I already knew how to add custom save actions. Didn't know about the template file. I override it and it worked like I needed it. Great success :)
